mne/viz/topomap.py

```diff
@@ -2382,3 +2382,171 @@ def _plot_corrmap(data, subjs, indices, ch_type, ica, label, show, outlines,
     fig.canvas.draw()
     plt_show(show)
     return fig
+
+
+def _trigradient(x, y, z):
+    """Take gradients of z on a mesh."""
+    from matplotlib.tri import CubicTriInterpolator, Triangulation
+    with warnings.catch_warnings():  # catch matplotlib warnings
+        warnings.filterwarnings("ignore", category=DeprecationWarning)
+        tri = Triangulation(x, y)
+        tci = CubicTriInterpolator(tri, z)
+        dx, dy = tci.gradient(tri.x, tri.y)
+    return dx, dy
+
+
+def plot_arrowmap(data, info_from, info_to=None, scale=1e-10, vmin=None,
+                  vmax=None, cmap=None, sensors=True, res=64, axes=None,
+                  names=None, show_names=False, mask=None, mask_params=None,
+                  outlines='head', contours=6, image_interp='bilinear',
+                  show=True, head_pos=None, onselect=None):
+    """Plot arrow map.
+
+    Compute arrowmaps, based upon the Hosaka-Cohen transformation [1]_,
+    these arrows represents an estimation of the current flow underneath
+    the MEG sensors. They are a poor man's MNE.
+
```

Comment:
Add this paragraph to the docstring to explain why one would want to use the `info_from` and `info_to` parameters:

```
# Since planar gradiometers takes gradients along latitude and longitude,
# they need to be projected to the flatten manifold span by magnetometer
# or radial gradiometers before taking the gradients in the 2D Cartesian
# coordinate system for visualization on the 2D topoplot. You can use the
# ``info_from`` and ``info_to`` parameters to interpolate from
# gradiometer data to magnetometer data.
```

Reply:
Fixed